classes/Utils.php

```diff
@@ -1120,6 +1120,26 @@ public function check_nonce() {
 		}
 	}
 
+	/**
+	 * Check if the openai response has any error or not.
+	 * If there any error then send the error response, otherwise continue.
+	 *
+	 * @since   3.0.0
+	 *
+	 * @param array $response The openai response.
+	 * @return mixed
+	 */
+	public function check_openai_response( array $response ) {
```

Comment:
Why this helper here? It's related to `tutorai` we can move it to `tutorai/Helper.php`

Reply:
I'll move it later. For now keep this as it is.